clustered/client/src/main/java/org/ehcache/clustered/client/internal/config/xml/ClusteringServiceConfigurationParser.java

```diff
@@ -128,7 +129,19 @@ public URI getNamespace() {
             /*
              * <read-timeout> is an optional element
              */
-            getTimeout = processGetTimeout(fragment, item);
+            getTimeout = processTimeout(fragment, item);
+
+          } else if ("mutative-timeout".equals(item.getLocalName())) {
+            /*
+             * <mutative-timeout> is an optional element
+             */
+            putTimeout = processTimeout(fragment, item);
```

Comment:
The variable names here should probably be consistent with the other naming (write|mutative).

Reply:
Yes. Done